downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,342 @@
+
+    public function main() returns error? {
+        float[] arr = [1.0, 2.0];
+        IntArray result = check arr.cloneWithType();
+        io:println(typeof result); // Prints 'typedesc IntArray'.
+    }
+    ```
+
+## Language updates
+
+### Bug fixes
+
+To view other bug fixes, see the [GitHub milestone for Swan Lake 2201.6.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.6.0+is%3Aclosed+label%3AType%2FBug).
+
+## Runtime updates
+
+### New features
+
+#### New Runtime Java APIs
+
+Introduced the `getInitMethod()` API in the `io.ballerina.runtime.api.types.ObjectType` class to get the method type of the initializer method of Ballerina objects.
+
+```java
+MethodType getInitMethod();
+```
+
+### Improvements
+
+#### Improvements in Runtime Java APIs
+
+The following APIs in the `io.ballerina.runtime.api` package are deprecated and marked for removal in a future release.
+
+  | **Runtime API**                                  | **Java class**                                     |
+  |--------------------------------------------------|----------------------------------------------------|
+  | `getCurrentRuntime()`                            | `io.ballerina.runtime.api.Runtime`                 |
+  | `createDecimalValue(String, DecimalValueKind)`   | `io.ballerina.runtime.api.creators.ValueCreator`   |
+  | `createStreamingJsonValue(JsonDataSource)`       | `io.ballerina.runtime.api.creators.ValueCreator`   |
+  | `convertToJson(Object, List<TypeValuePair>)`     | `io.ballerina.runtime.api.utils.JsonUtils`         |
+  | `getStringValue(Object, BLink)`                  | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `getExpressionStringValue(Object, BLink)`        | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `parseExpressionStringValue(String, BLink)`      | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `getValueKind()`                                 | `io.ballerina.runtime.api.values.BDecimal`         |
+  | `getStrand()`                                    | `io.ballerina.runtime.api.values.BFuture`          |
+  | `call(Strand, String, Object...)`                | `io.ballerina.runtime.api.values.BObject`          |
+  | `start(Strand, String, Object...)`               | `io.ballerina.runtime.api.values.BObject`          |
+  | `getRegExpDisjunction()`                         | `io.ballerina.runtime.api.values.BRegexpValue`     |
+  | `instantiate(Strand)`                            | `io.ballerina.runtime.api.values.BTypedesc`        |
+  | `instantiate(Strand, BInitialValueEntry[])`      | `io.ballerina.runtime.api.values.BTypedesc`        |
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.6.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
```

Comment:
We need to add a deprecation notice for the `serdes` package. How should we add it?

Reply:
We can add either as an improvement or a breaking change as done in [1] and [2].

[1] https://github.com/ballerina-platform/ballerina-dev-website/blob/6051eea53a87c869fabcfe29f90394a87684cb05/downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md?plain=1#L91
[2] https://github.com/ballerina-platform/ballerina-dev-website/blob/6051eea53a87c869fabcfe29f90394a87684cb05/downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md?plain=1#L300

Reply:
Shall we have a separate section, as discussed offline?

Reply:
Updated.